ios/Sources/TimetableFeature/List/TimetableContent.swift

```diff
@@ -0,0 +1,91 @@
+import ComposableArchitecture
+import Model
+import Styleguide
+import SwiftUI
+
+public struct TimetableContentState: Equatable {
+    public var items: [AnyTimetableItem]
+
+    public init(
+        items: [AnyTimetableItem]
+    ) {
+        self.items = items
+    }
+}
+
+public enum TimetableContentAction {
+    case tap(AnyTimetableItem)
+}
+
+public struct TimetableContentEnvironment {
+    public init() {}
+}
+
+public let timetableContentReducer = Reducer<TimetableContentState, TimetableContentAction, TimetableContentEnvironment> { _, _, _ in
+    .none
+}
+
+public struct TimetableContent: View {
+    private let store: Store<TimetableContentState, TimetableContentAction>
+
+    public init(store: Store<TimetableContentState, TimetableContentAction>) {
+        self.store = store
+    }
+
+    private let dateFormatter: DateFormatter = {
+       let formatter = DateFormatter()
+        formatter.setLocalizedDateFormatFromTemplate("HH:mm")
+        return formatter
+    }()
+
+    public var body: some View {
+        WithViewStore(store) { viewStore in
+            ScrollView {
+                LazyVStack(spacing: .zero) {
+                    ForEach(viewStore.items) { item in
+                        HStack(alignment: .top, spacing: 16) {
+                            Text(dateFormatter.string(from: item.startsAt))
+                                .font(.caption)
```

Comment:
when the content is top, font seems to be headline

Reply:
I don't know about this specification, so currently this specifies `caption` only.

top or currently living or etc.